modules/rosa-architecture.adoc

```diff
@@ -0,0 +1,13 @@
+[id="rosa-architecture_{context}"]
+= ROSA architecture on public and private networks
+
+You can install ROSA using either a public or private network.
+
+A private cluster and private network connection can be configured for ROSA during the cluster creation process.
+
```

Comment:
Consider this edit: You can configure a private cluster and network connection for ROSA when you create your cluster.

Reply:
Since I'm keeping "You can" in the first sentence, I made this sentence more active using different wording. Hope it suits, but lmk. Thanks :)